common/beerocks/bwl/include/bwl/mon_wlan_hal.h

```diff
@@ -65,7 +65,8 @@ class mon_wlan_hal : public virtual base_wlan_hal {
 
 // mon HAL factory types
 std::shared_ptr<mon_wlan_hal> mon_wlan_hal_create(std::string iface_name,
```

Comment:
```suggestion
std::shared_ptr<mon_wlan_hal> mon_wlan_hal_create(const std::string &iface_name,
```
While you're at it

Changes in this header file shall be done in the same commit as in the different bwl source code files

Reply:
fixed